src/utils/Downloader.js

```diff
@@ -80,6 +86,7 @@ class Downloader {
    * @param {string} opts.id Some id to later identify the task.
    * @param {number} opts.timeout Override global timeout
    * @param {boolean} opts.errorOn404 Treat 404 as error.
+   * @param {boolean} opts.forwardHeaders False to not forward the predefined request headers.
```

Comment:
I would invert the condition. specifying an explicit `false` is counter intuitive. 

or even better, turn in into an array with a default, eg:

```
@param {Array<string>} opts.forwardHeaders Names of headers that are forwarded. Default `['x-request-id', 'x-cdn-request-id', 'x-cdn-url']
```